XamarinCommunityToolkit/Behaviors/ImpliedOrderGridBehavior.shared.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Linq;
+using Xamarin.Forms;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.CommunityToolkit.Behaviors
+{
+	public class ImpliedOrderGridBehavior : Behavior<Grid>
```

Comment:
how about inheriting from BaseBehavior class?

Reply:
I looked at that, but I think I need Behavior<Grid> not Behavior<View> like that base class since it's only for a Grid and not any view.  Would it be worth making a generic base?

Reply:
Yeah, that definitely makes sense. One of PRs that is stuck now might bring it, but...uh
Probably I will make a PR with this change

Reply:
Ping me when that's in and I'll rebase and update.

Reply:
Merged